tests/test_sampling_initialization_failed.py

```diff
@@ -0,0 +1,54 @@
+"""Test sampling from a model where initialization will fail.
+
+Sampling from this model should generate ``Rejecting initial value`` logger
+messages followed by a C++ exception (which Cython turns into a Python
+exception). The exception is associated with the message ``Initialization
+failed.``
+"""
+import asyncio
+
+import aiohttp
+import pytest
+
+import helpers
+
+program_code = """
+parameters {
+  real y;
+}
+model {
+  y ~ uniform(100, 101);
+}
+"""
+
+@pytest.mark.asyncio
+async def test_sampling_initialization_failed(api_url: str) -> None:
+    """Test sampling from a model where initialization will fail."""
+    payload = {"function": "stan::services::sample::hmc_nuts_diag_e_adapt", "random_seed": 1}
```

Comment:
Will this fail always or is it just very probable (and should fail always due to seed)?

And could Stan3 changes affect this behaviour?

Reply:
Good question. It will always fail. By default, initial values are between -2 and 2 (sampled uniformly), see p. 57 in the CmdStan manual https://github.com/stan-dev/cmdstan/releases/download/v2.21.0/cmdstan-guide-2.21.0.pdf

I don't think stan3 will change anything.